tests/test_discretized_curves_space.py

```diff
@@ -26,17 +25,17 @@
                              initial_tangent_vec=INITIAL_TANGENT_VEC_C)
 
 N_SAMPLING_POINTS = 10
-SAMPLING_TIMES = gs.linspace(0, 1, N_SAMPLING_POINTS)
+SAMPLING_TIMES = gs.linspace(0., 1., N_SAMPLING_POINTS)
 DISCRETIZED_CURVE_A = CURVE_A(SAMPLING_TIMES)
 DISCRETIZED_CURVE_B = CURVE_B(SAMPLING_TIMES)
 DISCRETIZED_CURVE_C = CURVE_C(SAMPLING_TIMES)
 
 N_DISCRETIZED_CURVES = 5
-TIMES = gs.linspace(0, 1, N_DISCRETIZED_CURVES)
+TIMES = gs.linspace(0., 1., N_DISCRETIZED_CURVES)
 ATOL = 1e-8
 
 
-class TestDiscretizedCurvesSpaceMethods(unittest.TestCase):
+class TestDiscretizedCurvesSpaceMethods(geomstats.tests.TestCase):
     _multiprocess_can_split_ = True
 
     @classmethod
```

Comment:
Yes for the setUpClass, thanks.

Reply:
Still, what's the difference between declaring in the SetUpClass versus declaring constants at the top of the file?